.eslintrc.json

```diff
@@ -5,8 +5,10 @@
     "jest":true,
     "node":true
   },
-  "esversion":6,
-  "react/jsx-filename-extension": [1, { "extensions": [".js",".jsx"] }],
+  "rules":{
+    "react/jsx-filename-extension": [1, { "extensions": [".js",".jsx"] }],
+    "esversion":6
+  },
```

Comment:
Why did you make this change on this file?

Reply:
Oh yes, Actually the `esversion` and `react/jsz-filename-extension` cannot be in the root of the eslintrc file. It gives an error as seen from the hound review above.
```
- Unexpected top-level property "esversion".
```
They should be part of the `rules` in the eslintrc file.